src/component/1d/multiplicityTree/buildTreeNode.ts

```diff
@@ -8,12 +9,19 @@ import {
 
 import { TREE_LEVEL_COLORS } from './TreeColors';
 
-function createTreeNodes(signal: Signal1D, spectrumData) {
+export interface treeNodeData {
```

Comment:
```suggestion
export interface TreeNodeData {
```

Type names should always start with a capital letter